lib/protobuf/protoc/generator/util.ex

```diff
@@ -3,6 +3,8 @@ defmodule Protobuf.Protoc.Generator.Util do
 
   alias Protobuf.Protoc.Context
 
+  @locals_without_parens [field: 2, field: 3, oneof: 2, rpc: 3, extend: 4, extensions: 1]
```

Comment:
We already have these settings exported in `.formatter.exs`. Is there a simple way to reuse them without hacking Mix too much?

Reply:
Nothing worth it yet IMO 🙃